ItalianTranslation_PlanetCoaster.xml

```diff
@@ -4678,201 +4678,201 @@
+		<entry key="BuildingPartName_JUL_Flags_Bunting_Colored_Curve_4m" translation="Festone multicolore Curvo 4m"/>
+		<entry key="BuildingPartName_JUL_Flags_Bunting_Colored_Droop_2m" translation="Festone multicolore Ricurvo 2m"/>
+		<entry key="BuildingPartName_JUL_Flags_Bunting_Colored_Droop_4m" translation="Festone multicolore Ricurvo 4m"/>
+		<entry key="BuildingPartName_JUL_Flags_Bunting_Colored_Straight_2m" translation="Festone multicolore Dritto 2m"/>
+		<entry key="BuildingPartName_JUL_Flags_Bunting_Colored_Straight_4m" translation="Festone multicolore Dritto 4m"/>
+		<entry key="BuildingPartName_JUL_Flags_Large_Stand_IMG" translation="Bandiera con supporto grande personalizzabile"/>
+		<entry key="BuildingPartName_JU_BalloonCharacter_Eagle" translation="Personaggio Aquila Gonfiabile"/>
+		<entry key="BuildingPartName_JU_BalloonCharacter_UncleSam" translation="Personaggio Zio Sam Gonfiabile"/>
+		<entry key="BuildingPartName_JU_BalloonColumn_2m" translation="Colonna Gonfiabile 2m"/>
+		<entry key="BuildingPartName_JU_BalloonColumn_2m_Curve" translation="Colonna Gonfiabile Curva 2m"/>
+		<entry key="BuildingPartName_JU_Bunting_Straight_Curve_2m" translation="Festone Americano Curvo 2m"/>
+		<entry key="BuildingPartName_JU_Bunting_Straight_Curve_4m" translation="Festone Americano Curvo 4m"/>
+		<entry key="BuildingPartName_JU_Cannon" translation="Cannone per coriandoli"/>
+		<entry key="BuildingPartName_JU_Flags_Bunting_Droop_2m" translation="Festone Americano Ricurvo 2m"/>
+		<entry key="BuildingPartName_JU_Flags_Bunting_Droop_4m" translation="Festone Americano Ricurvo 4m"/>
+		<entry key="BuildingPartName_JU_Flags_Bunting_Straight_2M" translation="Festone Americano Dritto 2m"/>
+		<entry key="BuildingPartName_JU_Flags_Bunting_Straight_4m" translation="Festone Americano Dritto 4m"/>
+		<entry key="BuildingPartName_JU_Flags_Large_Hanging" translation="Bandiera Americana Appesa Grande "/>
+		<entry key="BuildingPartName_JU_Flags_Large_Hanging_IMG" translation="Bandiera Personalizzabile Appesa Grande"/>
+		<entry key="BuildingPartName_JU_Flags_Large_Stand" translation="Bandiera Americana con supporto Grande"/>
+		<entry key="BuildingPartName_JU_Flags_Single_Small" translation="Bandiera Americana con supporto Piccola"/>
+		<entry key="BuildingPartName_JU_Flags_Single_Small_IMG" translation="Bandiera Personalizzabile con supporto Piccola"/>
+		<entry key="BuildingPartName_PC_IMG_Mount_Ceiling" translation="Supporto Schermo da Soffitto"/>
+		<entry key="BuildingPartName_PC_IMG_Mount_Pole" translation="Supporto Schermo ad Asta"/>
+		<entry key="BuildingPartName_PC_IMG_Mount_Wall" translation="Supporto Schermo a Parete"/>
+		<entry key="BuildingPartName_PC_IMG_Projector" translation="Proiettore"/>
+		<entry key="BuildingPartName_PC_IMG_Scaffold" translation="Impalcatura di metallo"/>
+		<entry key="BuildingPartName_PC_IMG_Screen_Cinema_16m" translation="Schermo di Proiezione 16m - Acceso"/>
```

Comment:
Più che "acceso" secondo me ci sta meglio "illuminato", gli schermi sono tutti accesi
(Anche se ora non ricordo com'è in game)

Reply:
Si tratta di schermi accesi o spenti, da cui parte l'animazione video.. vedi tu, per me è indifferente.

Reply:
Allora per ora lascia pure acceso, modifichiamo in caso